requirements

```diff
@@ -0,0 +1,5 @@
+
+discord
+discord.py
+discord-py-slash-command
+python-dotenv
```

Comment:
Now that one *wasn't* intentional, I kept it in the project because there wasn't any need to remove it then before committing I was doing some tidying up, saw a file I wasn't using and removed it, forgetting that I didn't make that file lol.